src/styles/block/starts.scss

```diff
@@ -0,0 +1,25 @@
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  display: flex;
```

Comment:
Since `.stars__star` is a child element of `.stars` which is already a flex container, setting `display: flex;` again on the child is unnecessary unless you are planning to have flex items within `.stars__star` itself.